dhall/CHANGELOG.md

```diff
@@ -1,3 +1,118 @@
+1.25.0
+
+* Supports version 8.0.0 of the standard
```

Comment:
@Gabriel439 didn't you mean version 9.0.0?

Reply:
@ari-becker: Yes.  See: https://github.com/dhall-lang/dhall-haskell/issues/1168